### PR TITLE
Switch to React navigation

### PR DESCRIPTION
**Pull request by 4ian**

### CODE REVIEW

File: generators/base/templates/src/App.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
-import React from 'react';
-import Scenes from './Scenes';
+import { StackNavigator } from 'react-navigation';
 
-const App = () => (<Scenes />);
+import * as Pages from '<%= appName %>/src/pages';
 
-export default App;
+export default App = StackNavigator({
```
Comment (yleflour) [inline generators/base/templates/src/App.js]: Use a scenes file to store the stack

**Comment by yleflour** [comment]
Forbidding this merge unless someone does the following steps:
- Create a new React Native app
- Add Eslint
- Add Jest
- Add Base
- Run `yarn test`

**Comment by 4ian** [comment]
Ok will check that

**Comment by Nhacsam** [comment]
@4ian @Minishlink OK pour le run et le jest
eslint KO : 

/src/components/Button.js
  36:3  error  propType "children" is not required, but has no corresponding defaultProp declaration    react/require-default-props
  37:3  error  propType "onPress" is not required, but has no corresponding defaultProp declaration     react/require-default-props
  38:3  error  propType "buttonType" is not required, but has no corresponding defaultProp declaration  react/require-default-props

src/components/Page.js
  26:3  error  propType "children" is not required, but has no corresponding defaultProp declaration  react/require-default-props

src/pages/Home/index.js
  30:29  error  Multiple spaces found before '='  no-multi-spaces
src/pages/Infos/index.js
  29:29  error  Multiple spaces found before '='  no-multi-spaces

**Comment by 4ian** [comment]
Fixed eslint errors in the `base` generator :) I've also fixed the dependencies version in `eslint` generator, as I was getting errors with `eslint-plugin-jsx-a11y` (see https://github.com/bamlab/generator-rn-toolbox/pull/67/commits/d9dc214a9617fae6a7e2a2afa3735ff13620a7ed)

`yarn test` output:
```
yarn test v0.15.1
$ npm run test:lint && npm run test:unit

> GeneratorTest@0.0.1 test:lint /Users/florian/Projects/B/GeneratorTest
> eslint . --quiet


> GeneratorTest@0.0.1 test:unit /Users/florian/Projects/B/GeneratorTest
> jest

No tests found
  763 files checked.
  roots: /Users/florian/Projects/B/GeneratorTest - 763 matches
  testMatch: **/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x) - 83 matches
  testPathIgnorePatterns: /node_modules/ - 15 matches
✨  Done in 3.64s.
```

`react-native run-ios` screenshot:
image

**Comment by yleflour** [comment]
Merged in #70

**Pull request closed without merging**